packages/@aws-cdk/aws-stepfunctions/lib/states/parallel.ts

```diff
@@ -59,6 +60,16 @@ export interface ParallelProps {
    * @default - None
    */
   readonly resultSelector?: { [key: string]: any };
+
+  /**
+   * Parameters pass a collection of key-value pairs, either static values or JSONPath expressions that select from the input.
```

Comment:
Can we improve this doc string?  Something like

```suggestion
   * Parameters pass a collection of key-value pairs as input. Values can be literals or JSONPath expressions that select from the input.
```

What does "select from the input" mean?

Reply:
It means that it will select a part of the input to be used as value.
To be honest, this is a copy of the same property from the `Pass` step, but it always felt clear to me.